client/question/components/LearnerViewDirective.js

```diff
@@ -701,6 +759,12 @@ tie.directive('learnerView', [function() {
           $scope.nextButtonIsShown = false;
           var feedback = FeedbackObjectFactory.create();
           $scope.greetingParagraphs = feedback.getParagraphs();
+          EventHandlerService.createQuestionStartEvent(
+            SessionIdService.getSessionId(), $scope.currentQuestionId,
+            'QUESTION_VERSION');
```

Comment:
No -- declare QuestionDataService.getQuestionVersion(questionId) and return null from there for now (if we're working client-side), but add a TODO to it to make it actually return the correct question version once we surface this information from the server. Ditto elsewhere. Want to defend against any chance of these (invalid) strings getting stored.

Reply:
Why not QuestionObjectFactory.getQuestionVersion? Shouldn't that be stored on the object, rather than the service?

Reply:
SGTM.

Reply:
Had to make a version with no args since I couldn't avoid unused variables.